content/partner/aws.md

```diff
@@ -14,6 +14,6 @@ contact_us_form:
         name_title: Site Reliability Engineer, Cockroach Labs
         content: |
             We are building a distributed-database-as-a-service product that runs on Kubernetes clusters across
-            multiple public clouds including GCP, AWS and others. Pulumi's declarative model, the support for real
+            multiple public clouds including GCP, AWS and others. Pulumi's declarative model, the support for modern
```

Comment:
Quickly chatted w/ @isaac-pulumi  and I reverted the quotes back to the original text. We will need to reach out and get new quotes. Same for the other quote as well. Even though it might be fine given the person, it's probably better we treat all quotes equal.